dropwizard-logging/src/test/java/io/dropwizard/logging/FileAppenderFactoryTest.java

```diff
@@ -190,7 +190,10 @@ public void hasMaxFileSizeFixedWindow() throws Exception {
         RollingFileAppender<ILoggingEvent> appender = (RollingFileAppender<ILoggingEvent>) fileAppenderFactory.buildAppender(new LoggerContext());
 
         assertThat(appender.getRollingPolicy()).isInstanceOf(FixedWindowRollingPolicy.class);
+        assertThat(appender.getRollingPolicy().isStarted());
+        
         assertThat(appender.getTriggeringPolicy()).isInstanceOf(SizeBasedTriggeringPolicy.class);
+        assertThat(appender.getTriggeringPolicy().isStarted());
```

Comment:
The assertion was wrong, I needed to add an `isTrue`. This caused the test to fail before making the change.

Looks like this was added in logback-1.1.5[1] to make sure the triggering policy is started. Since the appenders are not started as part of the testing, the checks are never executed.

[1] https://github.com/qos-ch/logback/commit/1af14db4b6f28aee79f8f1bc866dfffe6cfdbe5a#diff-168f8cbb3bd8b03acc60eb147007d385R51